string INCREASE_LINEAR_VELOCITY = INCREASE_LINEAR_VELOCITY
string DECREASE_LINEAR_VELOCITY = DECREASE_LINEAR_VELOCITY
string INCREASE_ANGULAR_VELOCITY = INCREASE_ANGULAR_VELOCITY
string DECREASE_ANGULAR_VELOCITY = DECREASE_ANGULAR_VELOCITY
string TOGGLE_TARGET_FRAME_UP = TOGGLE_TARGET_FRAME_UP
string TOGGLE_TARGET_FRAME_DOWN = TOGGLE_TARGET_FRAME_DOWN
string TOGGLE_JOINT_UP = TOGGLE_JOINT_UP
string TOGGLE_JOINT_DOWN = TOGGLE_JOINT_DOWN
string TOGGLE_PLANE_UP = TOGGLE_PLANE_UP
string TOGGLE_PLANE_DOWN = TOGGLE_PLANE_DOWN
string TOGGLE_ROTATION_AXIS_UP = TOGGLE_ROTATION_AXIS_UP
string TOGGLE_ROTATION_AXIS_DOWN = TOGGLE_ROTATION_AXIS_DOWN

string[] pressed_commands
---
bool success
string error_msg